include/variables.h

```diff
@@ -389,107 +183,8 @@ extern u8 D_80157580[];
 extern u8 D_801579A0[];
 extern u64 gJpegUCodeData[];
 
-//extern ? D_8015BC30;
-//extern ? D_8015BC3C;
-//extern ? D_8015BC48;
-//extern ? D_8015BC58;
-//extern ? D_8015BC64;
-//extern ? D_8015BC70;
-//extern ? D_8015BC80;
-//extern ? D_8015BC8C;
-//extern ? D_8015BC98;
-//extern ? D_8015BCA8;
-//extern ? D_8015BCB4;
-//extern ? D_8015BCC0;
-//extern ? D_8015BCD0;
-//extern ? D_8015BCDC;
-//extern ? D_8015BCE8;
-//extern ? D_8015BCF8;
-//extern ? D_8015BCFC;
-//extern ? D_8015BD00;
-//extern ? D_8015BD08;
-//extern ? D_8015BD0A;
-//extern ? D_8015BD0C;
-//extern ? D_8015BD0E;
-//extern ? D_8015BD10;
-//extern ? D_8015BD34;
-//extern ? D_8015BD38;
-//extern ? D_8015BD3C;
-//extern ? D_8015BD50;
-//extern ? D_8015BD70;
-//extern ? D_8015BD7C;
-//extern ? D_8015BD80;
-//extern ? D_8015BD84;
-//extern ? D_8015BD90;
-//extern ? D_8015BD9C;
-//extern ? D_8015CE46;
-//extern ? playerFloorPoly;
-//extern ? D_8015CE58;
-//extern ? D_8015CE80;
-//extern ? D_8015CE8C;
-//extern ? D_8015CEA0;
-//extern ? D_8015CEA8;
-//extern ? D_8015CEC0;
-//extern ? D_8015CED0;
-//extern ? D_8015CF00;
-//extern ? D_8015CF10;
-//extern ? D_8015CF12;
-//extern ? D_8015CF14;
-//extern ? D_8015D3A4;
-//extern ? D_8015D3A8;
-//extern ? D_8015D3CC;
-//extern ? D_8015D3D0;
-//extern ? D_8015D3D8;
-//extern ? D_8015D3DA;
-//extern ? D_8015D3DC;
-//extern ? D_8015D86C;
-//extern ? D_8015D870;
-//extern ? D_8015D894;
-//extern ? D_8015D898;
-//extern ? D_8015D8A0;
-//extern ? D_8015D8A2;
-//extern ? D_8015D8A4;
-//extern ? D_8015DD34;
-//extern ? D_8015DD38;
-//extern ? D_8015DD5C;
-//extern ? D_8015DD60;
-//extern ? D_8015DD68;
-//extern ? D_8015DD6A;
-//extern ? D_8015DD6C;
-//extern ? D_8015E1FC;
-//extern ? D_8015E200;
-//extern ? D_8015E224;
-//extern ? D_8015E228;
-//extern ? D_8015E230;
-//extern ? D_8015E268;
-//extern ? D_8015E2A0;
-//extern ? D_8015E2D8;
-//extern ? D_8015E310;
-//extern ? D_8015E320;
-//extern ? D_8015E358;
-//extern ? D_8015E390;
-//extern ? D_8015E3A0;
-//extern ? D_8015E3D8;
-//extern ? D_8015E410;
-//extern ? D_8015E420;
-//extern ? D_8015E430;
-//extern ? D_8015E440;
-//extern ? D_8015E478;
-//extern ? D_8015E4B0;
-//extern ? D_8015E4C0;
-//extern ? D_8015E4F8;
-//extern ? D_8015E530;
-//extern ? D_8015E564;
-//extern ? D_8015E598;
-//extern ? D_8015E5A8;
-//extern ? D_8015E5DC;
-//extern ? D_8015E610;
-//extern ? D_8015E61C;
-//extern ? D_8015E628;
-//extern ? D_8015E638;
-//extern ? D_8015E648;
-extern SaveContext gSaveContext; // 0x8015E660
-extern GameInfo* gGameInfo; // 0x8015FA90
+extern SaveContext gSaveContext;
+extern GameInfo* gGameInfo;
```

Comment:
I think its good keeping the addresses here.

Reply:
we're about to start supporting other versions soon, it doesnt really make sense to.

Save context you can look up with sym info. game info is a little harder cause its dynamically allocated but its right after the gamestate instance. and you rarely need to look up the address for it (in my experience)

Reply:
right but until we do I dont see a need to take it out.

Reply:
the information is easily available and will be irrelevant soon, i dont see it as an issue to remove now

Reply:
btw the address that was commented near `gGameInfo` is the address of the `gGameInfo` symbol, not the typical location of the `GameInfo` instance

regardless, I don't think it's a good idea to put "typical allocated addresses" in the source